frontend/src/components/TableWithModal/index.tsx

```diff
@@ -72,6 +69,7 @@ export const TableWithModal = ({
         handleRowDoubleClick={handleRowDoubleClick}
         handleRowClick={handleRowClick}
         handleTableNewButton={handleTableNewButton}
+        isReadOnly={canEdit}
```

Comment:
just as an observation the isReadOnly flag logic seems flawed.. meaning if canEdit = true, means isReadOnly should be false, maybe your logic is sound... but your semantics are off.